random-beans/src/main/java/io/github/benas/randombeans/util/FastClasspathScannerFacade.java

```diff
@@ -48,8 +49,7 @@
 
     static {
         typeToConcreteSubTypes = new ConcurrentHashMap<>();
-        // disable blacklisting of JRE system jars and system packages (java.* and sun.*)
-        scanResult = new FastClasspathScanner("!!").scan();
+        scanResult = new FastClasspathScanner().unBlacklistSystemPackages().whitelistPackages("*").scan();
```

Comment:
@PascalSchumacher I think you're missing either `.enableClassInfo()`  or `.enableAllInfo()`

Reply:
That was what I thought too, but `whitelistPackages` calls `enableClassInfo`.

Reply:
Ah interesting, I've only skimmed over the documentation haven't integrated it yet.

Reply:
Correct, I made `.whitelistPackages()` call `.enableClassInfo()`, since it seemed to make sense to do so -- or is it confusing to have that as the default?

However, you don't need `.whitelistPackages("*")` -- if no whitelist is provided, all packages are whitelisted for recursive scanning. I will make this more clear in the docs.